pkg/translate/translate.go

```diff
@@ -13,6 +14,11 @@ import (
 
 type TranslateServiceServer struct {
 	pb.UnimplementedTranslateServiceServer
+	repo repo.Repo
+}
```

Comment:
As far as I understood repo field, holds a copy of repo, not the repo instance itself,
Maybe it should hold a pointer to `repo.Repo` ?
```go
type TranslateServiceServer struct {
	pb.UnimplementedTranslateServiceServer
	repo *repo.Repo
}
```